index.js

```diff
@@ -3,12 +3,15 @@ const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
-/***** Deliverable 2 *****/
 
+/***** Deliverable 2 *****/
+const newHeader = document.querySelector("#header")
+console.log(newHeader.style.color = 'blue')
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const firstCardImg = document.querySelector('div#profile img')
```

Comment:
Take this step by step. You have the firstCardImg. It's an `image` element. That means it has an `src` tag. How can we access the `src` tag to feed it a URL?

Take this same approach for the other elements and their properties! An `h2` for example, has `textContent`